backend/src/flotilla/api/events_api.py

```diff
@@ -62,11 +63,24 @@ async def post_event(
 ) -> Event:
     """Add a new event to the robot schedule"""
     try:
+        end_time = event_request.start_time + DEFAULT_EVENT_DURATION
+        overlapping_events: List[Event] = read_events_by_robot_id_and_time_span(
+            db=db,
+            robot_id=event_request.robot_id,
+            start_time=event_request.start_time,
+            end_time=end_time,
+        )
+        if overlapping_events:
+            raise HTTPException(
+                status_code=HTTPStatus.CONFLICT.value,
+                detail=f"Conflict with already existing event in the same time period. Events with id: {','.join(str(event.id) for event in overlapping_events)}",
```

Comment:
Will all users be authorized to see all events for a given robot?
If not, this might be giving out too much information, maybe the first statement is enough?
I'm not sure, open for discussion